src/store/CategoricalDisplayStore.ts

```diff
@@ -346,6 +346,7 @@ export default class CategoricalDisplayStore {
     }
 
     handleFilterClick(categoryId: string): void {
+        this.focusedCategoryKey = '';
```

Comment:
This clear up was added in order to trigger component change in case when user clicked twice and > on same category.